packages/editor-ui/src/components/ParameterInputHint.vue

```diff
@@ -15,18 +18,3 @@ export default Vue.extend({
 	},
 });
 </script>
-
-<style scoped lang="scss">
-.hint {
-	margin-top: var(--spacing-2xs);
```

Comment:
is margin not needed anymore? it's anyways it's better to have the margin in the parent component instead of here.

Reply:
input has some margin, it was redundant